eng/psakefile.ps1

```diff
@@ -74,9 +74,31 @@ Task parser -Depends init {
 }
 
 Task pyqir-tests -Depends init {
-    Invoke-LoggedCommand -workingDirectory (Join-Path $repo.root pyqir-tests) {
-        & $python -m pip install tox
-        & $python -m tox -e test
+    $srcPath = $repo.root
+    $project = "pyqir-tests"
+    $installationDirectory = Resolve-InstallationDirectory
+
+    if (Test-RunInContainer) {
+        Build-ContainerImage $srcPath
+        Write-BuildLog "Running container image:"
+        $ioVolume = "$($srcPath):/io"
+        $llvmVolume = "$($installationDirectory):/usr/lib/llvm"
+        $userSpec = ""
+
+        Invoke-LoggedCommand {
+            docker run --rm $userSpec -v $ioVolume -v $llvmVolume -e LLVM_SYS_110_PREFIX=/usr/lib/llvm -w /io/$project manylinux2014_x86_64_maturin conda run --no-capture-output python -m tox -e test
+        }
+    }
+    else {
+        exec -workingDirectory (Join-Path $srcPath $project) {
+            Invoke-LoggedCommand {
+                & $python -m pip install tox
+            }
```

Comment:
I don't know PowerShell but using `&` inside of  `Invoke-LoggedCommand` seems redundant, compared to the lines 88-90.

Reply:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_Operators?view=powershell-7.2#call-operator-

It is used because $python is a variable whereas docker above is a bare command